lib/engine/api/client.rb

```diff
@@ -14,6 +14,8 @@ def get_api(url, config, api_key, params = {})
 
           config.logger.debug("GET #{url}") if config.debug_enabled
         end
+      rescue StandardError => e
+        Logger.new('log/api_client.log').info("#{e}\nURL:#{url}\nparams:#{params}")
```

Comment:
@maksimf can we use the already set up logger instead of a new file? So customer know where to expect it.